core/init.php

```diff
@@ -22,12 +22,8 @@
     die('$page variable is unset. Cannot continue.');
 }
 
-if (!file_exists(ROOT_PATH . '/core/config.php')) {
-    if (is_writable(ROOT_PATH . '/core')) {
-        fopen(ROOT_PATH . '/core/config.php', 'w');
-    } else {
-        die('Your <strong>/core</strong> directory is not writable, please check your file permissions.');
-    }
+if (!is_writable(ROOT_PATH)) {
```

Comment:
Why the whole `ROOT_PATH` here? Shouldn't we be more specific for just the config path?

Reply:
I forgot is_writable obviously doesn't check recursively